Django-1.10: Fix esmon.api import error

# esmond_manage migrate
Traceback (most recent call last):
  File "esmond/manage.py", line 14, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/lib/python2.7/dist-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/usr/lib/python2.7/dist-packages/django/core/management/__init__.py", line 341, in execute
    django.setup()
  File "/usr/lib/python2.7/dist-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/lib/python2.7/dist-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/lib/python2.7/dist-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/dist-packages/esmond/api/__init__.py", line 7, in <module>
    from esmond.api.models import OIDSet
  File "/usr/lib/python2.7/dist-packages/esmond/api/models.py", line 5, in <module>
    from django.contrib.auth.models import User, Permission
  File "/usr/lib/python2.7/dist-packages/django/contrib/auth/models.py", line 4, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/usr/lib/python2.7/dist-packages/django/contrib/auth/base_user.py", line 52, in <module>
    class AbstractBaseUser(models.Model):
  File "/usr/lib/python2.7/dist-packages/django/db/models/base.py", line 105, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/usr/lib/python2.7/dist-packages/django/apps/registry.py", line 237, in get_containing_app_config
    self.check_apps_ready()
  File "/usr/lib/python2.7/dist-packages/django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")

--- a/esmond/api/__init__.py
+++ b/esmond/api/__init__.py
@@ -4,7 +4,6 @@
 """
 
 from esmond.config import get_config_path, get_config
-from esmond.api.models import OIDSet
 
 # Prefix used in all the snmp data cassandra keys
 SNMP_NAMESPACE = 'snmp'
@@ -28,6 +27,7 @@
 
     def generate_endpoint_map(self):
         payload = {}
+        from esmond.api.models import OIDSet
         for oidset in OIDSet.objects.all().order_by('name'):
             for oid in oidset.oids.all().order_by('name'):
                 if oid.endpoint_alias:
